### DISCUSSION TITLE
Can't generate SCSS sourceMaps

### DISCUSSION THREAD

**Discussion by james0r** [uicrooks/shopify-theme-lab#115] [Q&A]
What's the cleanest way to implement sourceMaps for SCSS? not able to generate them by following the webpack sass-loader instructions.

thanks!

**Comment by sergejcodes** [comment]
Sourcemaps with Shopify only work with `devtool: 'inline-source-map'` and `style-loader` instead of `MiniCssExtractPlugin.loader` so the final CSS won't be extracted to a separate file and instead is loaded via JavaScipt.

But with the current way, the Shopify CLI handles HMR and reloading, if you would add source handling you would lose the reload/HMR functionality because Shopify expects CSS file changes.

**Reply by james0r** [reply]
Yeah I kinda figured this out while playing around. I ended up just compiling my scss into css chunks for sections/components/pages and thus don't so much need sourcemaps for the time being.
